app/models/application_reference.rb

```diff
@@ -35,8 +35,6 @@ class ApplicationReference < ApplicationRecord
     never_asked: 'never_asked',
   }
 
-  scope :minimum_feedback_provided?, -> { where(feedback_status: 'feedback_provided').count >= 2 }
```

Comment:
There seem to be lots of places and ways that we check whether an application has had the correct number of references (just spotted while looking through this PR). Is it worth raising a tech debt ticket to make sure it's all done in one place?

Reply:
Yes, this would be good. We used to allow references to be returned after the candidate had submitted, so I wouldn't be surprised if there were some checks hanging around.

Reply:
Raised https://trello.com/c/Yku5746x/428-duplicate-check-for-number-of-references-for-an-application-form